components/SponsorCard.vue

```diff
@@ -0,0 +1,60 @@
+<template>
+  <a class="sponsor-logo" :href="url">
+    <img :src="`_nuxt/assets/img/sponsors/${tier}/${name}.png`" :alt="name" />
+  </a>
+</template>
+
+<script lang="ts">
```

Comment:
I would use Vue 3 composition API here, using `<script setup lang="ts">`. This makes it so that you can define props a lot easier. Something like this:

```
defineProps<{
  name: string;
  tier: string;
  url: string;
}>();
```